aidants_connect_web/tests/test_functional/test_404.py

```diff
@@ -1,24 +1,15 @@
-from django.contrib.staticfiles.testing import StaticLiveServerTestCase
-from selenium.webdriver.firefox.webdriver import WebDriver
 from django.test import tag
 
+from .testcases import FunctionalTestCase
```

Comment:
Par atavisme, j'aime quand les imports sont très très explicites 
```suggestion
from aidants_connect_web.tests.test_fonctional.testcases import FunctionalTestCase
```

Reply:
Aucun souci, je les repasse en absolu 😄